.evergreen/install-dependencies.sh

```diff
@@ -96,7 +96,7 @@ fi
 
 if [[ $operating_system != "win" ]]; then
   # Update npm to latest when we can
-  npm install --global npm@latest
+  npm install --global npm@9
```

Comment:
TODO we'll sync with the real fix when it is merged

Reply:
done!